### ISSUE TITLE
verdi devel tests asks for sudo

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida_core#2121]
reported by @lekah @sphuber 
It seems, the `verdi profile delete` test in `aiida.backend.tests.cmdline.commands.profile` causes a request for sudo to delete the DB.

**Comment by ltalirz** [comment]
One solution could be to add flags --delete-db/--no-delete-db that allow to skip the step of deleting the DB.

Opinions?

**Comment by dev-zero** [comment]
:+1: for `--delete-db/--no-delete-db`

**Comment by ltalirz** [comment]
I just wanted to fix this, adding the needed options and cleaning up `verdi profile delete` a bit.
When it came to running the tests, I noticed that I am unable to reproduce the reported behavior with current `develop`:

```
verdi -p test_django devel tests db.cmdline.commands.profile
vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
>>> Tests for django db application
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
.....
----------------------------------------------------------------------
Ran 5 tests in 0.472s

OK
Final summary of the run of tests:
* Tests skipped: 0
* Tests run:     5
* Tests failed:  0
* Tests errored: 0
```

Can someone confirm that the behavior on their machine is different and let me know the exact OS & AiiDA version please?

**Comment by sphuber** [comment]
Confirmed:

OS: Ubuntu 16.04.5
DB: Postgres 9.5
AiiDA: develop [commit 1337bc2a5b1a80579030e8f8be665b035e7c1c69](https://github.com/aiidateam/aiida_core/commit/1337bc2a5b1a80579030e8f8be665b035e7c1c69)

**Comment by espenfl** [comment]
Confirmed:

OS: CentOS 7.4
DB: Postgres 9.2
AiiDA: develop commit 1337bc2a5b1a80579030e8f8be665b035e7c1c69

**Comment by ltalirz** [comment]
Thanks guys ;-) I'll look into it later today

**Comment by sphuber** [comment]
Fixed in PR #2151
